static/app/components/charts/styles.tsx

```diff
@@ -13,8 +13,7 @@ export const SubHeading = styled('h3')`
 `;
 
 export const SectionHeading = styled('h4')`
-  display: inline-grid;
-  grid-auto-flow: column;
+  display: flex;
   gap: ${space(1)};
```

Comment:
Are we officially using flex with gap now?

Reply:
I think we are, yeh. I remember us arguing about this a while ago @evanpurkhiser

Reply:
Okay, we’re not using flex + gap. I remembered our argument wrong 😅